jwks_test.go

```diff
@@ -28,9 +29,12 @@ const (
+var (
+	// jwksJSON is a embedded JWKS in JSON format.
+	//go:embed example_jwks.json
+	jwksJSON string
```

Comment:
Removing the need to keep the example_jwks.json and this value in sync.

Reply:
Glad to see this. It would have been best to include that in this release: https://github.com/MicahParks/keyfunc/releases/tag/v1.2.2